storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -705,6 +705,13 @@
      */
     @isString(acceptedValues = { "S0", "S1", "S2", "S3" })
     public static final String TOPOLOGY_LOGGING_SENSITIVITY = "topology.logging.sensitivity";
+    /**
+     * Log file the user can use to configure Log4j2.
```

Comment:
Nit: Consider mentioning that the configuration file is applied in addition to the regular config file, using these rules https://logging.apache.org/log4j/2.x/manual/configuration.html#CompositeConfiguration.